Sunrise/Helpers/AppRouting.swift

```diff
@@ -3,18 +3,38 @@
 //
 
 import UIKit
+private func < <T : Comparable>(lhs: T?, rhs: T?) -> Bool {
+  switch (lhs, rhs) {
+  case let (l?, r?):
+    return l < r
+  case (nil, _?):
+    return true
+  default:
+    return false
+  }
+}
```

Comment:
That is a leftover from automatic migration that I missed to remove among all these files. Good catch 👍 ☺️